README.md

```diff
@@ -57,6 +57,24 @@ export BACKEND_URL_publisher=http://localhost:3001
 
 Routes reference these backends by their ID (e.g., "frontend", "publisher").
 
+### Serving routes from a flat file
+
+When `ROUTER_ROUTES_FILE` is set, Router will load routes from the specified JSONL file (one JSON object per line):
```

Comment:
Can you link to the JSONL spec/website/whatever. I don't think it's that common. I had to look it up.